### ISSUE TITLE
Opera 12.x fails to open connection

### ISSUE THREAD

**Issue by trygve-lie** [einaros/sse.js#5]
Opera 12.x seems to fail connecting due to incorrect Content-Type "text/x-dom-event-stream".  Opera 12.x now seems to handle Content-Type "text/event-stream" just fine these days.

The main problem is that the regex for handling older versions of Opera which used the incorrect Content-Type "text/x-dom-event-stream" seems to kick in on newer versions of Opera causing them to fail. Though current version of Opera is 12.x and the versions wanting the incorrect Content-Type is pre version 10.x. I'm pretty sure there isn't many pre 10.x versions of Opera out there anymore so a complete removal of the "isLegacy" checks can probably be done.

There is also a small issue in Opera after the Content-Type is set to "text/event-stream" causing it to disconnect. Opera does not seem to happy about getting:

'my message'\n
\n

It seems to want the end of a message to be like this:

'my message'\n\n

In other words, this will cause Opera to disconnect on a message:

``` javascript
this.res.write('data: ' + line + '\n');
this.res.write('\n');
```

It should be written like this on the last line of a message:

``` javascript
this.res.write('data: ' + line + '\n\n');
```

PS: If you agree that removing the "isLegacy" checks is a good idea I can submit a pullrequest that fixes both this issues since I've already have fixed this in a fork I've made.
